
--- glpk/examples/Makefile.in	2013-01-28 01:00:00.000000000 -0800
+++ glpk.OLD/examples/Makefile.in	2019-06-11 10:35:54.910948733 -0700
@@ -70,7 +70,7 @@
 am_glpsol_OBJECTS = glpsol.$(OBJEXT)
 glpsol_OBJECTS = $(am_glpsol_OBJECTS)
 glpsol_LDADD = $(LDADD)
-glpsol_DEPENDENCIES = ../src/libglpk.la
+glpsol_DEPENDENCIES = ../src/libcoinglpk.la
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
@@ -207,7 +207,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 INCLUDES = -I$(srcdir)/../src
-LDADD = ../src/libglpk.la
+LDADD = ../src/libcoinglpk.la
 glpsol_SOURCES = glpsol.c
 all: all-am
 
@@ -278,6 +278,7 @@
 	  sed -e 'h;s,^.*/,,;s/$(EXEEXT)$$//;$(transform)' \
 	      -e 's/$$/$(EXEEXT)/' `; \
 	test -n "$$list" || exit 0; \
+	{ test -d "$(DESTDIR)$(bindir)" && test -x "$(DESTDIR)$(bindir)"; } || exit 0; \
 	echo " ( cd '$(DESTDIR)$(bindir)' && rm -f" $$files ")"; \
 	cd "$(DESTDIR)$(bindir)" && rm -f $$files
 

--- glpk/src/glplpx01.c	2019-06-11 10:37:36.472364887 -0700
+++ glpk.OLD/src/glplpx01.c	2019-06-11 10:35:54.912948721 -0700
@@ -520,7 +520,7 @@
       ret = glp_simplex(lp, &parm);
       switch (ret)
       {  case 0:           ret = LPX_E_OK;      break;
-         case GLP_EBADB:
+         case GLP_EBADB:   ret = LPX_E_BADB;    break;
          case GLP_ESING:
          case GLP_ECOND:
          case GLP_EBOUND:  ret = LPX_E_FAULT;   break;

--- glpk/src/Makefile.in	2013-01-28 01:00:00.000000000 -0800
+++ glpk.OLD/src/Makefile.in	2019-06-11 10:37:20.857454659 -0700
@@ -262,10 +262,10 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-include_HEADERS = glpk.h
-lib_LTLIBRARIES = libglpk.la
+include_HEADERS = glpk.h amd/amd.h
+lib_LTLIBRARIES = libcoinglpk.la
 libglpk_la_LDFLAGS = -version-info 33:0:0 \
--export-symbols-regex '^(glp_|_glp_lpx_).*'
+-export-symbols-regex '^(glp_|_glp_lpx_|_glp_amd_defaults|_glp_amd_order).*'
 
 libglpk_la_SOURCES = \
 glpapi01.c \
@@ -467,7 +467,7 @@
 	  echo rm -f $${locs}; \
 	  rm -f $${locs}; \
 	}
-libglpk.la: $(libglpk_la_OBJECTS) $(libglpk_la_DEPENDENCIES) $(EXTRA_libglpk_la_DEPENDENCIES) 
+libcoinglpk.la: $(libglpk_la_OBJECTS) $(libglpk_la_DEPENDENCIES) $(EXTRA_libglpk_la_DEPENDENCIES) 
 	$(libglpk_la_LINK) -rpath $(libdir) $(libglpk_la_OBJECTS) $(libglpk_la_LIBADD) $(LIBS)
 
 mostlyclean-compile:
